demo/basic.html

```diff
@@ -0,0 +1,149 @@
+<!doctype html>
```

Comment:
Can we do this in a different pull request? There is a bit more work involved in reorganizing demos.